exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/DatabaseConfig.kt

```diff
@@ -85,26 +86,28 @@ class DatabaseConfig private constructor(
          * Useful when [eager loading](https://github.com/JetBrains/Exposed/wiki/DAO#eager-loading) is used.
          */
         var keepLoadedReferencesOutOfTransaction: Boolean = false,
-
         /**
          * Set the explicit dialect for a database.
          * This can be useful when working with unsupported dialects which have the same behavior as the one that
          * Exposed supports.
          */
         var explicitDialect: DatabaseDialect? = null,
-
         /**
          * Set the default schema for a database.
          */
         var defaultSchema: Schema? = null,
-
         /**
          * Log too much result sets opened in parallel.
          * The error log will contain the stacktrace of the place in the code where a new result set occurs, and it
          * exceeds the threshold.
          * 0 value means no log needed.
          */
         var logTooMuchResultSetsThreshold: Int = 0,
+        /**
+         * Toggle whether table and column identifiers that are also keywords should retain their case sensitivity.
+         * Keeping user-defined case sensitivity (value set to `true`) will become the default in future releases.
+         */
+        var preserveKeywordCasing: Boolean = true,
```

Comment:
Based on some feedback from users, I've added a global flag to `DatabaseConfig`.

I'd appreciate any thoughts on:
- Whether it should default to true or false
- Whether it should be released with a deprecation warning or if that should come in a future release
- How long the deprecation cycle should be for this flag
- Or whether this flag should even be used at all.

Reply:
Would it not ideally default to `false` in the beginning so we have time to declare it in the release notes and ease users into it?